.travis.yml

```diff
@@ -9,16 +9,19 @@ jobs:
       stage: test
       compiler: clang
       python: 2.7
+      node_js: 7
       addons:
         apt:
           sources: ['ubuntu-toolchain-r-test']
-          packages: ['cmake', 'nodejs', 'g++-5']
+          packages: ['cmake', 'g++-5']
       before_install:
         - export CC="${CC_COMPILER}"
         - export CXX="${CXX_COMPILER}"
         - export ASAN_OPTIONS="symbolize=1"
       install:
         - pip install --user flake8
+        - "curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash -"
```

Comment:
@kripken How would I use a polyfill here? Also, the files I added to the tests are very large. I suspect it would be better to have a separate smaller test case for this functionality and not check in large test files for the actual unit tests like those in i32.wast.